pkg/esp32_sdk/patches/0018-bootloader-changes-for-esp32c3.patch

```diff
@@ -0,0 +1,79 @@
+From ca93a21d6e2bc58d1d32da594351bb03f94343a0 Mon Sep 17 00:00:00 2001
+From: Gunar Schorcht <gunar@schorcht.net>
+Date: Thu, 10 Mar 2022 12:38:51 +0100
+Subject: [PATCH 18/20] bootloader: changes for esp32c3
```

Comment:
What are those changes / why are they needed?
Future readers will need this information.